### PR TITLE
[5.9.0] - Fix JSON web key set response

### PR DESCRIPTION
**Pull request by deshankoswatte**
## Purpose
- Add the proper response obtained from the JSON web key set URL.
- Fixes: https://github.com/wso2/product-is/issues/13329.

### CODE REVIEW

File: en/docs/learn/json-web-key-set-endpoint.md [hunk 1]
```diff
@@ -44,7 +44,15 @@ specification](http://openid.net/specs/openid-connect-discovery-1_0.html)
 <a class="sourceLine" id="cb1-7" title="7">      <span class="st">&quot;kid&quot;</span>: <span class="st">&quot;NTAxZmMxNDMyZDg3MTU1ZGM0MzEzODJhZWI4NDNlZDU1OGFkNjFiMQ&quot;</span>,</a>
 <a class="sourceLine" id="cb1-8" title="8">      <span class="st">&quot;alg&quot;</span>: <span class="st">&quot;RS256&quot;</span>,</a>
 <a class="sourceLine" id="cb1-9" title="9">      <span class="st">&quot;n&quot;</span>: <span class="st">&quot;luZFdW1ynitztkWLC6xKegbRWxky-5P0p4ShYEOkHs30QI2VCuR6Qo4Bz5rTgLBrky03W1GAVrZxuvKRGj9V9-PmjdGtau4CTXu9pLLcqnruaczoSdvBYA3lS9a7zgFU0-s6kMl2EhB-rk7gXluEep7lIOenzfl2f6IoTKa2fVgVd3YKiSGsyL4tztS70vmmX121qm0sTJdKWP4HxXyqK9neolXI9fYyHOYILVNZ69z_73OOVhkh_mvTmWZLM7GM6sApmyLX6OXUp8z0pkY-vT_9-zRxxQs7GurC4_C1nK3rI_0ySUgGEafO1atNjYmlFN-M3tZX6nEcA6g94IavyQ&quot;</span></a>
-<a class="sourceLine" id="cb1-10" title="10">    }</a>
+<a class="sourceLine" id="cb1-10" title="10">    },</a>
```
Comment (nilminiwso2) [inline en/docs/learn/json-web-key-set-endpoint.md:47]: We seem to have duplicated the section. Was this intentional?

Screenshot 2022-06-20 at 12 53 05

Comment (deshankoswatte) [reply en/docs/learn/json-web-key-set-endpoint.md:47]: This is intentional. If you take a look at a response of that endpoint from the WSO2 Identity Server 5.9.0, it is as follows.
image

Comment (deshankoswatte) [reply en/docs/learn/json-web-key-set-endpoint.md:47]: I might need to change the column values, will do it now.

Comment (deshankoswatte) [reply en/docs/learn/json-web-key-set-endpoint.md:47]: The above said is fixed with https://github.com/wso2/docs-is/pull/3059/commits/8f22f74b66eb0fa0be1bd20d4c0e680f9946a680.

Comment (nilminiwso2) [reply en/docs/learn/json-web-key-set-endpoint.md:47]: ack.

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**